### ISSUE TITLE
Image preview partially broken

### ISSUE THREAD

**Issue by CosmosJunk** [0x-jerry/vscode-hexo-utils#59]
**Describe the bug**
Images inserted with hexo "asset_img" syntax not working in preview.

**To Reproduce**
1. Edit markdown file in vscode, with vscode-hexo-utils enabled
2. 'ctrl+shift+p' and select '>markdown: open preview to the side'
3. {% asset_img blahblah %} inserted images won't show properly.

**Expected behavior**
Images properly shown in preview tab.

**Screenshots**
![hexopreviewbug2](https://user-images.githubusercontent.com/76647894/191913169-3e4834db-e8b2-4b78-96b0-bb23c75d3c60.png)

![screencapture-localhost-4000-2022-09-23-16_01_00](https://user-images.githubusercontent.com/76647894/191916575-72bb5166-adc1-4787-a3a6-201c36a3a80d.png)

**Plugin version(please complete the following information):**
v0.3.0

**Additional context**
I am pretty new to everything (vscode, hexo, etc. ). I've re-installed vscode and disabled most of the other extensions.
A hexo boot on localhost:4000/ works fine. Vscode preview tab not showing the following ones.
```
{% asset_img "twilight_s" "twilight_s.webp" 600 400 "冷战热斗" "alt twi_s" %}

{% asset_img "twilight_amazon_panel.jpg" "冷战热斗面板" %}
```

**Environment**
OS: Windows11 21H2
VS Code: 1.71.2
Node.js: 16.14.2

Activated extensions:
Chinese Language pack
Hexo Utils v0.3.0
Paste Image v1.0.4

Hexo under Ubuntu WSL2:
hexo: 6.3.0
hexo-cli: 4.3.0
os: linux 5.10.102.1-microsoft-standard-WSL2 Ubuntu 20.04 LTS (Focal Fossa)
node: 16.17.0

**Comment by 0x-jerry** [comment]
Fixed, please upgrade to v0.3.1 and try again.

Thanks for your contribution!

**Comment by CosmosJunk** [comment]
Thank you! Can confirm v0.3.1 works fine.

Cheers.
